src/main/java/bisq/price/spot/providers/CoinGecko.java

```diff
@@ -87,15 +109,49 @@ public Set<ExchangeRate> doGet() {
                             this.getName()
                     ));
                 });
-
+        this.addSpeciallyConsideredCurrencies(result, rates);
         return result;
     }
 
+    /**
+     * Whenever possible, add currency exchange rates for currencies that are generally excluded from bisq
+     *
+     * @param result where to add the exchange rate to
+     * @param rates obtained from coingecko
+     */
+    private void addSpeciallyConsideredCurrencies(Set<ExchangeRate> result, Map<String, CoinGeckoTicker> rates) {
+        try {
+            Double officialArsBtcRate = rates.get(CoinGecko.ARS_EXCHANGE_RATE_KEY).getValue().doubleValue();
+            Double blueArsBtcRate = this.getBlueGapMultiplier() * officialArsBtcRate;
+            this.log.debug("Current ARS/USD blue/official gap: " + blueArsBtcRate);
+            result.add(new ExchangeRate(
+                    CoinGecko.ARS_EXCHANGE_RATE_KEY,
+                    BigDecimal.valueOf(blueArsBtcRate),
+                    new Date(),
+                    this.getName()
+            ));
+        } catch (Exception e) {
+            this.log.warn("Something went wrong when adding ARS/BTC: " + e.getMessage(), e);
+        }
+    }
+
+    private Double getBlueGapMultiplier() {
+        return Objects.requireNonNull(restTemplate.exchange(
+                RequestEntity
+                        .get(UriComponentsBuilder
+                                .fromUriString(CoinGecko.GET_USD_EXCHANGE_RATES_ARG_URL).build()
```

Comment:
Making a secondary unrelated HTTP request within coingecko provider is unclean and could break the coingecko feed at runtime.  It could be done more cleanly by making the bluelytics feed an `ExchangeRateProvider` and doing the special rate conversion at the `ExchangeRateService` level.

Reply:
> Making a secondary unrelated HTTP request within coingecko provider is unclean and could break the coingecko feed at runtime. It could be done more cleanly by making the bluelytics feed an `ExchangeRateProvider` and doing the special rate conversion at the `ExchangeRateService` level.

@jmacxx  I agree, I think I'll move this last commit to a separate branch to work on later on and we can merge this PR without this last change if you guys agree